libbeat/outputs/tidb_slow_query/client.go

```diff
@@ -0,0 +1,233 @@
+package tidb_slow_query
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/elastic/beats/v7/libbeat/logp"
+	"github.com/elastic/beats/v7/libbeat/outputs"
+	"github.com/elastic/beats/v7/libbeat/publisher"
+	"github.com/go-sql-driver/mysql"
+	lru "github.com/hashicorp/golang-lru"
+	"time"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+const (
+	// 2000 clusters
+	insertStmtCacheSize = 2000
+	noTable             = 1146
+	noPartition         = 1526
```

Comment:
what are they?

Reply:
Mysql's error number.

1146 means no table exists. When a new TiDB cluster is created, generates the first slow log, and filebeat try to insert this log, this error will be triggered. Filebeat will try to create new table for the new cluster.

1526 means no partition. This error occurs when the latest partition boundary < timestamp of incoming slow log. Filebeat will try to create new partition.

Reply:
Please change either the name or add comments to make sure that others can know that this is an error number of MySQL.